src/db/user_in_group.js

```diff
@@ -5,7 +5,7 @@ const database = require('../database')
 
 const UserInGroup = class {
   constructor (userInGroup) {
-    this._id = userInGroup.id
+    this._id = userInGroup.group_id
```

Comment:
`group_id`(alone) is not a PK. This file has (and has allways had) sigificant problems. I suggest deleting it. Only the the `add_User` function is ever used (never the class), this should be moved to `db/group`.